### ISSUE TITLE
Web version

### ISSUE THREAD

**Issue by burningSchadow42** [SMARTMarkers/smartmarkers-ios#1]
Is this porject available as a general web app?
iOS and MacOS are not common enough in my part of the world for this porject to be usefull otherwise.

**Comment by raheelsayeed** [comment]
You are right. For now iOS support is active. However there is a `React` build -->  `https://github.com/SMARTMarkers/smartmarkers-react-patient-app`, but requires updating. 

These can run as web apps too. But may need some updating.
